{"name": "困人处置列表-真实", "status": "passed", "attachments": [{"name": "log", "source": "59ebb835-78e9-42d6-b6df-b9a484048492-attachment.txt", "type": "text/plain"}], "start": 1690436276571, "stop": 1690436276812, "uuid": "712fc18d-624b-445b-be89-83b4420eea33", "historyId": "2c9991b785716977eb4cbbaa1f323277", "testCaseId": "792deb5b60e48cc51a654ad1e9657ccb", "fullName": "opsPC.test_emergency#test_device_error_trapped_list_003", "labels": [{"name": "story", "value": "应急处置"}, {"name": "feature", "value": "运维管理平台"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_emergency"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "10244-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_emergency"}]}